kazoo/recipe/lock.py

```diff
@@ -267,6 +291,56 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.release()
 
 
+def WLock(*args, **kwargs):
+    """Kazoo read-write lock (writer side).
+
+    Example usage:
+
+    .. code-block:: python
+
+        zk = KazooClient()
+        lock = WLock("/lockpath", "my-identifier")
```

Comment:
Obviously I don't know the code base, but the creation and non-use of `zk` looks fishy to me. Is there an implicit global client?